spec/controllers/catalog_controller_spec.rb

```diff
@@ -8,7 +8,7 @@
     @item = instantiate_fixture(@druid, Dor::Item)
     @user = User.find_or_create_by_webauth double('WebAuth', :login => 'sunetid', :logged_in? => true, :attributes => {'DISPLAYNAME' => 'Example User'}, :privgroup => '')
     allow(Dor).to receive(:find).with("druid:#{@druid}").and_return(@item)
-    allow(Dor::Item).to receive(:find).with("druid:#{@druid}").and_return(@item)
+    allow(Dor).to receive(:find).with("druid:#{@druid}").and_return(@item)
```

Comment:
since it's an allow, and the two lines are now identical, could we just remove this?